File: Examples/SwiftCoreiOSExample-ObjC/SwiftCoreiOSExample-ObjC/ViewController.m
//
//  ViewController.m
//  SwiftCoreiOSExample-ObjC
//
//  Created by Brian Nickel on 3/1/23.
//

#import "ViewController.h"
@import HeapSwiftCore;

@interface ViewController ()
@end

@implementation ViewController

- (IBAction)buttonClicked:(id)sender
{
    [Heap.sharedInstance track:@"Button Clicked" properties:@{
        @"source language": @"Objective C ❤️",
    }];
}

@end

File: Examples/SwiftCoreMacExample/SwiftCoreMacExample/ViewController.swift
//
//  ViewController.swift
//  SwiftCoreMacExample
//
//  Created by Jerry Jones on 9/13/22.
//

import Cocoa
import HeapSwiftCore

class ViewController: NSViewController {

    override func viewDidLoad() {
        
        super.viewDidLoad()
    }
    
    @IBAction func butonClicked(_ sender: Any?) {
        
        HeapLogger.shared.debug("👍 Button was clicked, and tracked",
                                   source: "SwiftCoreMacExample")
        Heap.shared.track("Button Clicked")
    }
}

